cmake.deps/cmake/TreesitterCMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.10)
 project(tree-sitter C)
-
+if(NOT MSVC)
+  add_compile_options(-std=c99)
+endif()
```

Comment:
Try

```suggestion
set(CMAKE_C_STANDARD 99)
```

and see if that solves it.

Reply:
Yes it works.

Reply:
TreesitterParserCMakeLists.txt already have this option.